mcs/build/tests.make

```diff
@@ -96,8 +96,10 @@ endif
 
 test_assemblies :=
 
+test_lib_output = $(topdir)/class/lib/$(PROFILE)/$(test_lib)
```

Comment:
this won't work, a lot of the tests look up resources and other test stuff relative to the test assembly location and won't find those files anymore.